src/store/status/getters.ts

```diff
@@ -21,4 +28,31 @@ export const getters: GetterTree<TxStatus, RootState> = {
     const status = state.txDetails ? state.txDetails.status : '';
     return setStatusMessage(state.type, status);
   },
+  [constants.STATUS_GET_ESTIMATED_RELEASE_TIME_IN_MINUTES]: (state: TxStatus)
+    : Promise<number> => new Promise<number>((resolve, reject) => {
+      const bridgeService = new BridgeService();
+      const web3 = new Web3(EnvironmentAccessorService.getEnvironmentVariables().vueAppRskNodeHost);
+      if (state.txDetails) {
+        const status = state.txDetails as PegoutStatusDataModel;
+        web3.eth.getTransaction(status.originatingRskTxHash)
+          .then(({ blockNumber }) => {
+            if (blockNumber) reject(new Error('The tx are not mined yet'));
```

Comment:
If block number exists then it means that transaction is not mined yet? Sounds like it is quite the opposite, could be?

Reply:
same question here